modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -285,6 +312,8 @@ private function sorting_to_wp_query_param( $sort ) {
 	function update( $new_instance, $old_instance ) {
 		$instance = array();
 
+		error_log( print_r( array( $_GET, $_POST ), true ) );
```

Comment:
I don't think you meant to leave it here :)

Reply:
Fixed